docs/advanced/language-extensions.md

```diff
@@ -70,10 +70,14 @@ Example:
 <SideBySide>
 
 <!-- prettier-ignore -->
+
 ```ts
-for (const i of $range(1, 5)) {}
-for (const i of $range(1, 10, 2)) {}
-for (const i of $range(5, 1, -1)) {}
+for (const i of $range(1, 5)) {
```

Comment:
Why change this? And if you were to change it, at least also do the same for the lua on the right, so the two blocks still line up.

Reply:
The autoformatter did this... I'm turning it off